osu.Game/Screens/Play/Player.cs

```diff
@@ -365,6 +366,15 @@ private void load(AudioManager audio, OsuConfigManager config, OsuGameBase game,
 
             IsBreakTime.BindTo(breakTracker.IsBreakTime);
             IsBreakTime.BindValueChanged(onBreakTimeChanged, true);
+
+            skipIntroOverlay.IsSkippable.ValueChanged += e =>
+            {
+                if (Configuration.AutomaticallySkipIntro && e.NewValue && RestartCount > 0)
+                {
+                    Configuration.AutomaticallySkipIntro = false;
```

Comment:
Why? `Configuration` should not be modified without good reason.

Reply:
If the user made a quick restart, the intro would be skipped automatically as expected. However if the user wants to restart the map again without skipping the intro this would not be the case. The intro would be skipped on each restart regardless of quick restart or normal restart. By setting the bool afterwards to false does not break the expected behavior.

Reply:
I think you're misunderstanding this data structure. It's meant to be immutable after a use. It's created once for each `Player`.